easypost/__init__.py

```diff
@@ -291,21 +294,26 @@ def request_raw(self, method, url, params=None, apiKeyRequired=True):
 
         if hasattr(ssl, 'OPENSSL_VERSION'):
             ua['openssl_version'] = ssl.OPENSSL_VERSION
-
-        headers = {
-            'X-Client-User-Agent': json.dumps(ua),
-            'User-Agent': USER_AGENT,
-            'Authorization': 'Bearer %s' % my_api_key,
-            'Content-type': 'application/x-www-form-urlencoded'
-        }
-
+        if isRating:
```

Comment:
no need to recreate the dict a second time. could just create the headers, and then replace the key.
```
headers = {...}
if isRating:
    headers['Content-Type'] = 'application/json'
```